<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInputDlgBarcode</class>
 <widget class="QDialog" name="CInputDlgBarcode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eingabe-Barcode</string>
  </property>
  <widget class="QSpinBox" name="spinBoxCount">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Barcode:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBarcode">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserArticleInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>471</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Artikelnummer:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxArticlenumber">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBoxCount</tabstop>
  <tabstop>comboBoxArticlenumber</tabstop>
  <tabstop>lineEditBarcode</tabstop>
  <tabstop>textBrowserArticleInfo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
